### PR TITLE
global: allow utf8 in commit message

### PR DESCRIPTION
**Pull request by helix84**
* E.g. the Signed-off-by below contains diacritics and will pass
   "kwalitee check message" after this commit.

Signed-off-by: Ivan Masár <helix84@centrum.sk>

### CODE REVIEW

**Comment by helix84** [comment]
Still getting these from "kwalitee check message", I don't really understand what seems to be wrong with the bullet:

3: M102 unrecognized bullet/signature
4: M102 unrecognized bullet/signature

**Comment by jirikuncar** [comment]
@helix84 you have over indented the bullet point.

Here are suggestions for an improvement:
```
global: fix problem with UTF-8 in commit message

* Allows UTF-8 in commit message e.g. the `Signed-off-by` ...

Signed-off-by: ...
```

**Comment by helix84** [comment]
@jirikuncar Thanks, fixed. I spent too much time on trying to figure out what's wrong there. Perhaps we should make M102 clearer, too.

**Comment by helix84** [comment]
I added a test case but it always passes. How do I get it to catch the exception? (something like UnicodeEncodeError: 'ascii' codec can't encode character u'\xe1' in position 215: ordinal not in range(128))

**Comment by jirikuncar** [comment]
@helix84 it might depend on the Python version. You can try to force a bytestream with `b'* Příliš žluťoučký kůň úpěl ďábelské ódy.'` when it is passed to `check_message`.

**Comment by helix84** [comment]
I was testing just check_message(), but that wasn't what was failing.  The problem was printing in click, so I added another test there.

**Comment by jirikuncar** [comment]
@helix84 you can ignore the QuantifiedCode hook.

**Comment by helix84** [comment]
@jirikuncar ping

File: tests/cli/test_check.py [hunk 1]
```diff
@@ -81,3 +81,14 @@ def test_check_branch_wrong_side(capsys, git):
     result = runner.invoke(check, ['-r', git, 'message', 'testbranch..master'])
     assert_that(result.exit_code, equal_to(0))
     assert_that(result.output.split("\n"), equal_to([""]))
+
+
+@skip
```
Comment (jirikuncar) [inline tests/cli/test_check.py]: Do you really want to skip it?

Comment (helix84) [inline tests/cli/test_check.py]: Yes, same as the other tests in that file. Here's why:
https://github.com/helix84/kwalitee/blob/31c309917e08371c4621d420f262bf2ebbd917ab/tests/cli/test_check.py#L39-L40

Comment (jirikuncar) [inline tests/cli/test_check.py]: Hups. Thanks!

**Pull request merged**